lib/ash/resource/change/cascade_destroy.ex

```diff
@@ -309,9 +313,11 @@ defmodule Ash.Resource.Change.CascadeDestroy do
           |> Ash.load!(
             [
               {relationship.name,
-               Ash.Query.set_context(relationship.destination, %{cascade_destroy: true})}
+               Ash.Query.set_context(relationship.destination, %{
+                 cascade_destroy: true,
+                 accessing_from: %{source: relationship.source, name: relationship.name}
+               })}
             ],
-            authorize?: false,
```

Comment:
only the loads used `authorize?: false` the bulk_<update/destroy>s don't. In my mind it's better to just set the accessing_from everywhere and let the policies do their thing?